ml-data-hub-plugin/src/main/groovy/com/marklogic/gradle/DataHubPlugin.groovy

```diff
@@ -45,13 +41,13 @@ import org.springframework.context.annotation.AnnotationConfigApplicationContext
 @EnableAutoConfiguration
 class DataHubPlugin implements Plugin<Project> {
 
-    private DataHub dataHub
-    private Scaffolding scaffolding
-    private HubProject hubProject
-    private HubConfig hubConfig
+    private DataHubImpl dataHub
+    private ScaffoldingImpl scaffolding
+    private HubProjectImpl hubProject
+    private HubConfigImpl hubConfig
     private LoadHubModulesCommand loadHubModulesCommand
     private LoadUserStagingModulesCommand loadUserStagingModulesCommand
-    private MappingManager mappingManager
+    private MappingManagerImpl mappingManager
     private FlowManagerImpl flowManager
```

Comment:
This really defeats the point of us having interfaces in the core. But not sure why we have them anyways given we need to create an object for everything anyways.